tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -20,37 +20,37 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Pull nginx
-    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  nginx
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  ${nginx}
 
 Pull busybox
-    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  busybox
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  ${busybox}
 
 Pull ubuntu
-    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  ubuntu
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  ${ubuntu}
 
 Pull non-default tag
-    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  nginx:alpine
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  ${nginx}:${alpine}
```

Comment:
Can't do that. The colon designates a tag on the right side.  So this is the nginx image using the alpine tag, which is not on the harbor instance currently.